samples/system-test/subscriptions.test.ts

```diff
@@ -19,34 +19,36 @@ import {
   Topic,
 } from '@google-cloud/pubsub';
 import {assert} from 'chai';
-import {describe, it, afterEach} from 'mocha';
+import {describe, it, after} from 'mocha';
 import {execSync, commandFor} from './common';
 import * as uuid from 'uuid';
 
 describe('subscriptions', () => {
   const projectId = process.env.GCLOUD_PROJECT;
   const pubsub = new PubSub({projectId});
   const runId = uuid.v4();
-  const topicNameStem = `stest-topic-${runId}`;
-  const subNameStem = `stest-sub-${runId}`;
-
-  let nextTopicId = 0;
-  async function createTopic(): Promise<Topic> {
-    const id = `${topicNameStem}-${nextTopicId++}`;
+  console.log(`Subscriptions runId: ${runId}`);
```

Comment:
Is this `console.log` intention?

Reply:
Yep! It's for debug purposes in case there are test failures or leftover topics. (We do it in several of the test suites, so should be okay here.)